notification/twilio/voice.go

```diff
@@ -471,13 +438,13 @@ func (v *Voice) ServeTest(w http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	resp := newTwiMLResponse(w)
+	resp := newTwiMLResponse(ctx, w)
 	switch call.Digits {
 	default:
 		resp.SayUnknownDigit()
 		fallthrough
 	case "", digitRepeat:
-		resp.Say(call.msgBody)
+		processSayBody(resp, call.msgBody, call.msgPauseIndex)
```

Comment:
This feels like it should just be a method

Reply:
I can do it if you want, although I do think since it's voice specific, so I will make it part of the Voice object.